astropy/coordinates/builtin_frames/equatorial.py

```diff
@@ -65,21 +65,20 @@ class TETE(BaseRADecFrame):
     Calculation of the true equator and equinox involves the application of both precession
     and nutation, whilst only applying precession gives a mean equator and equinox.
 
-    TETE coordinates are often referred to as "apparent" coordinates, although CIRS
-    has replaced TETE as the modern system to use for apparent coordinates. TETE is the
-    apparent coordinate system used by JPL Horizons and is the correct coordinate
-    system to use when combining the right ascension with local apparent sidereal time
-    to calculate the hour angle.
+    TETE coordinates are often referred to as "apparent" coordinates, or
+    "apparent place". TETE is the apparent coordinate system used by JPL Horizons
+    and is the correct coordinate system to use when combining the right ascension
+    with local apparent sidereal time to calculate the hour angle.
```

Comment:
Oh, one more point. Hour angle there should read TIRS hour angle. The observed (ITRS) hour angle is slightly different due to the effects of polar motion.

Reply:
Given that polar motion is ~0.3 arcsecs I think I will leave out this distinction. My motivation is to keep the docstring as simple as possible so that astropy users don't try LST = HA + RA with ICRS coordinates (which I have seen in the wild several times).